modules/wkt/docs/api-reference/wkb-writer.md

```diff
@@ -0,0 +1,57 @@
+# WKBLoader
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.2-blue.svg?style=flat-square" alt="From-v3.1" />
+</p>
+
+Writer for the [Well-known binary][wkb] format for representation of geometry.
+
+[wkb]: https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary
+
+| Loader                | Characteristic                                |
+| --------------------- | --------------------------------------------- |
+| File Extension        | `.wkb`,                                       |
+| File Type             | Binary                                        |
+| File Format           | [Well Known Binary][wkb]                      |
+| Data Format           | [Geometry](/docs/specifications/category-gis) |
+| Supported APIs        | `encode`, `encodeSync`                        |
+| Encoder Type          | Synchronous                                   |
+| Worker Thread Support | Yes                                           |
+
+## Usage
+
+```js
+import {WKBWriter} from '@loaders.gl/wkt';
+import {encodeSync} from '@loaders.gl/core';
+
+const geometry = {
+  type: "Polygon",
+  coordinates: [[[1, 2], [3, 4], [5, 6], [1, 2]]]
+}
+const arrayBuffer = encodeSync(geometry, WKBWriter, {wkt: {hasZ: false, hasM: false}})
+```
+
+## Options
+
+- `hasZ`: Should be `true` if the GeoJSON input has Z values. These values are expected to be the third coordinate position.
```

Comment:
As I recall, these were not really options in WKX but fields on their base geometry class. I turned them into options as the simplest way forward..

Do we normally autodetect these for other formats? Should we try to do that longer term also for WKB?

Reply:
WKB has different concepts for Z and M (measure) coordinates. By having these as options, if your data has 3 dimensions you could declare the 3rd dimension to be and M coordinate, not a Z coordinate. Shapefile I believe also has this distinction, but when parsing to GeoJSON we don't record what dimension the 3rd coordinate value is